esmvaltool/recipes/recipe_multimodel_products_wp5.yml

```diff
@@ -0,0 +1,80 @@
+#
+# Description
+# Namelist to compute the ensemble mean anomaly and the ensemble
+# variance and agreement and plot the results as maps and time series.
+#
+# Authors
+# Nicolaur Manubens (BSC)
+# Alasdair Hunter (BSC)
+#
+#
+# Project
+# KNMI Magic
+#
+# Reference
+# Manubens, Nicolau, Louis-Philippe Caron, Alasdair Hunter, Omar Bellprat,
+# Eleftheria Exarchou, Neven S. Fučkar, Javier Garcia-Serrano et al.
+# "An R package for climate forecast verification." Environmental
+# Modelling & Software 103 (2018): 29-42.
+#
+#
+# This namelist is part of the ESMValTool
+###############################################################################
+---
+
+datasets:
+  - {dataset: bcc-csm1-1, type: exp, project: CMIP5,  mip: Amon,  exp: historical,  ensemble: r1i1p1,  start_year: 1961,  end_year: 1990}
+#  - {dataset: GFDL-ESM2G, type: exp, project: CMIP5,  mip: Amon,  exp: historical,  ensemble: r1i1p1,  start_year: 2000,  end_year: 2002}
+  - {dataset: MPI-ESM-MR, type: exp, project: CMIP5,  mip: Amon,  exp: historical,  ensemble: r1i1p1,  start_year: 1961,  end_year: 1990}
+  - {dataset: MIROC5,     type: exp, project: CMIP5,  mip: Amon,  exp: historical,  ensemble: r1i1p1,  start_year: 1961,  end_year: 1990}
+  - {dataset: MPI-ESM-MR, type: exp, project: CMIP5,  mip: Amon,  exp: rcp26,  ensemble: r1i1p1,  start_year: 2006,  end_year: 2099}
+  - {dataset: bcc-csm1-1, type: exp, project: CMIP5,  mip: Amon,  exp: rcp26,  ensemble: r1i1p1,  start_year: 2006,  end_year: 2099}
+  - {dataset: MIROC5,     type: exp, project: CMIP5,  mip: Amon,  exp: rcp26,  ensemble: r1i1p1,  start_year: 2006,  end_year: 2099}
+
+#latlon_roi: (/latmin,latmax,lonmin,lonmax/)
+preprocessors:
+  preproc:
+    regrid:
+      target_grid: bcc-csm1-1
+      scheme: linear
+    mask_fillvalues:
+      threshold_fraction: 0.95
+    extract_region:
+      start_longitude: 0
+      end_longitude: 360
+      start_latitude: -90
+      end_latitude: 90
+
+diagnostics:
+  anomaly_agreement:
+    description: Calculate multi-member anomalies and their spread/agreement.
+    variables:
+      tas:
+        preprocessor: preproc
+        mip: Amon
+        field: T2M
+    scripts:
+      main:
+        script: magic_bsc/multimodel_products.r #(in diag_scripts)
+
+        #Specify datasets to be used for computing climatology and anomaly
+        climatology_class: historical
+        anomaly_class: rcp26
+
+        climatology_start_year: 1961
+        climatology_end_year: 1990
+
+        anomaly_start_year: 2006
+        anomaly_end_year: 2099
+        #Parameters for Season() function
+        moninf: 6   #If this is null, then the monthly anomalies will be computed
+        monsup: 6
+
+        agreement_threshold: 80
+        #Time series plot options
+        running_mean: 5 #Length of running mean to use for the time series plot
+
+        # Timeseries plot
+        time_series_plot: NULL # Either mean or maxmin (plot the mean with/without shading between the max and min.
```

Comment:
If I run this diagnostic with `time_series_plot: maxmin`, it crashes with the following error:
```
Loading required package: maps

Attaching package: ‘s2dverification’

The following object is masked from ‘package:base’:

    Filter


Attaching package: ‘ClimProjDiags’

The following object is masked from ‘package:s2dverification’:

    Subset

Warning message:
In dir.create(run_dir, recursive = TRUE) :
  '/group_workspaces/jasmin2/cp4cds1/magic-test/tmp-bandela/MAGIC_BSC/output_dir/recipe_multimodel_products_wp5_time_series_plot_maxmin_20190124_155958/run/anomaly_agreement/main' already exists
Saving 7 x 7 in image
Warning messages:
1: Removed 12 rows containing non-finite values (stat_summary). 
2: Removed 12 rows containing missing values (geom_path). 

 *** caught bus error ***
address 0x7faa8562f090, cause 'non-existent physical address'

Traceback:
 1: axis(2, at = ypos, labels = ylabs, cex.axis = cex_axes_labels,     tcl = cex_axes_ticks, mgp = c(0, spaceticklab + 0.2, 0))
 2: PlotEquiMap(data, lat = lat, lon = lon, brks = brks, units = units,     toptitle = title, filled.continents = FALSE, dots = drop(agreement) >=         agreement_threshold, fileout = paste0(plot_dir, "/",         var0, "_", months, "_multimodel-anomaly_", start_anomaly,         "_", end_anomaly, "_", start_climatology, "_", end_climatology,         ".png"))
An irrecoverable exception occurred. R is aborting now ...
```

Reply:
Dear @bouweandela ,
I've corrected the names of the options to plot the time series. The user can select between 'single' which plot a individual time series for each model and 'maxmin' which plot a shadow from the minimum to the maximum value.